### PR TITLE
this is a test

### PR DESCRIPTION
**Pull request by hww228**

### CODE REVIEW

**Comment by tillahoffmann** [comment]
It looks like one of the additional clones of the shedding-hub repository may have snuck in here. Theres a section "Submodule shedding-hub added at 8fd08e" under the changed files [here](https://github.com/shedding-hub/shedding-hub/pull/49/files#diff-4d42dc904b05c8ff996dee796b1ddf47c53e258574ac1698689bafcb5e251b8c).

**Comment by YWAN446** [comment]
Youwei, I assume the submodule was created when you cloned another "shedding-hub" within the directory of the "shedding-hub". We can review your codes together with Till this Friday. Meanwhile, I will just go ahead and make comments related to your YAML files directly.

File: vetter2020daily/vetter2020.csv [hunk 1]
```diff
@@ -0,0 +1,75 @@
+patient number,dpo,Date,Sample,Log copies/ml,Virus isolation,PatientID,Sex,Age
```
Comment (YWAN446) [inline vetter2020daily/vetter2020.csv:1]: I am sorry you may have mentioned this in our previous meetings. But how did you create vetter2020.csv? Is that from the Challenger data?

File: shedding-hub
Comment (YWAN446) [inline shedding-hub:1]: We will take a look at your git setup together during our meeting on Friday. I assume you are pushing the first layer of git while changes in the second layer of git. Or the other around. Till should be able to help.

**Review by YWAN446 — COMMENTED** [review]
Here are some initial comments.

File: vetter2020daily/Vetter2020Daily.yaml [hunk 2]
```diff
@@ -0,0 +1,497 @@
+# yaml-language-server: $schema=.schema.yaml
+title: 'Daily Viral Kinetics and Innate and Adaptive Immune Response Assessment in
+  COVID-19: a Case Series'
+doi: https://doi.org/10.1128/msphere.00827-20
+description: >-
```
Comment (YWAN446) [inline vetter2020daily/Vetter2020Daily.yaml:5]: For `folded` block, to remove the `-` after `>`. You need to add "\n" at the end of your string within the block. I will add another comment in `vetter2020daily.py.

**Pull request closed without merging**